### ISSUE TITLE
[Landroid V4] missing codes

### ISSUE THREAD

**Issue by sihui62** [nibi79/worxlandroid#91]
OH version: openHAB 4.0.0 Build #3544
Binding version: 4.0.0.202307021742
Worx Mower M500 Plus WR165E

`WorxLandroidStatusCodes.java` is missing 3 status codes:

```
ESCAPE_FROM_OLM (13, "Escape from OLM")
ZONE_TRAINING (97, "Zone Training")
MANUAL_STOP (99, "Manual Stop")
```

`WorxLandroidErrorCodes.java` is missing 1 error code:

`MOWER_OUTSIDE_WIRE (20, "Mower outside wire")`

**Comment by clinique** [comment]
Ok for these codes, but ZONE_TRAINING is also present with code 31. WDYT ?

**Comment by sihui62** [comment]
> but ZONE_TRAINING is also present with code 31

Yes, I think so. If it stops raining I will do a test next week  :-)

**Comment by TomW80** [comment]
Hello, 

I missing the Status "Mowing process".
My Landroid is Mowing now and I get only "NO_ERR"

**Comment by Boby71** [comment]
> Hello,
> 
> I missing the Status "Mowing process". My Landroid is Mowing now and I get only "NO_ERR"
> 
> I use org.openhab.binding.worxlandroid-4.1.0-SNAPSHOT.jar

Aren't you confusing here STATUS CODE and ERROR CODE?

**Comment by TomW80** [comment]
Yes, I've noticed that in the meantime too
